relay/api/app.py

```diff
@@ -74,6 +75,8 @@ def add_resource(resource, url):
     add_resource(ExchangeAddresses, '/exchange/exchanges')
     add_resource(UnwEthAddresses, '/exchange/eth')
     add_resource(OrderDetail, '/exchange/order/<string:order_hash>')
+    add_resource(UserEventsExchange, '/exchange/<address:exchange_address>/users/<address:user_address>/events')
```

Comment:
The other endpoints, for example `/exchange/orderbook` do not separate the different exchanges. Maybe we should then also not do it for the events? Not sure about this, but usually the idea is that it should not matter which exchange was used.

Reply:
Hmm yes I get your point...  My take was to stay with our other event endpoints though:
- `/networks/<address:network_address>/users/<address:user_address>/events`
- `/tokens/<address:token_address>/users/<address:user_address>/events`

Reply:
Agreed. So we can probably just have both.